x-pack/test/performance/scalability/config.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrConfigProviderContext } from '@kbn/test';
+import fs from 'fs';
+import path from 'path';
+import { serializeApmGlobalLabels } from '../utils';
+import { ScalabilityTestRunner } from './runner';
+
+// These "secret" values are intentionally written in the source.
+const APM_SERVER_URL = 'https://142fea2d3047486e925eb8b223559cae.apm.europe-west1.gcp.cloud.es.io';
+const APM_PUBLIC_TOKEN = 'pWFFEym07AKBBhUE2i';
+const AGGS_SHARD_DELAY = process.env.LOAD_TESTING_SHARD_DELAY;
+const DISABLE_PLUGINS = process.env.LOAD_TESTING_DISABLE_PLUGINS;
+
+export default async function ({ readConfigFile, log }: FtrConfigProviderContext) {
+  const performanceConfig = await readConfigFile(require.resolve('../journeys/base.config.ts'));
+
+  const scalabilityJsonPath = process.env.SCALABILITY_JOURNEY_PATH;
+  const isJsonPathDefined =
+    scalabilityJsonPath &&
+    fs.existsSync(scalabilityJsonPath) &&
+    path.extname(scalabilityJsonPath) === '.json';
+
+  const apmGlobalLabels = {
+    ...performanceConfig.get('kbnTestServer').env.ELASTIC_APM_GLOBAL_LABELS,
+    journeyFilePath: isJsonPathDefined ? path.basename(scalabilityJsonPath) : 'unknown',
+    journeyName: isJsonPathDefined
+      ? JSON.parse(fs.readFileSync(scalabilityJsonPath, 'utf8')).journeyName
+      : 'unknown',
+  };
```

Comment:
So far we will have these 4 APM labels to filter data:

```
{
  "testJobId":"local-b260d025-75f2-4943-b84d-0af319344630",
  "testBuildId":"local-5d7ccda7-e3e6-43be-8b00645a86ce744f",
  "journeyFilePath":"promotion_tracking_dashboard-0182cff0-e8d8-42f1-b3c5-1f2e71dc7862.json",
  "journeyName":"promotion_tracking_dashboard"
}
```

Reply:
IMHO we can start with the status quo and amend it later if we need it.